components/ProjectsScrollPagination/ProjectsScrollPagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { useSWRPages } from 'swr'
+import React from 'react'
+
+import ActivityIndicator from '~/components/ActivityIndicator'
+import { SearchProjectsParams } from '~/common/api-endpoints'
+import { useSWRWithExtras } from '~/hooks/use-swr'
+import ProjectCard from '~/components/ProjectCard'
+import { APIEndpoint } from '~/common'
+
+interface IProjectsScrollPagination {
+  filters?: SearchProjectsParams
+}
+
+function ProjectsScrollPagination({ filters }: IProjectsScrollPagination) {
```

Comment:
```suggestion
function ProjectScrollPagination({ filters }: ProjectScrollPaginationProps) {
```

Só um conserto de nomeação em inglês.